From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Wed, 7 May 2025 19:05:20 +0200
Subject: Fix name of not-installed test
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Guido Günther <agx@sigxcpu.org>
---
 tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index 91a6fdc..62e89a4 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -6,7 +6,7 @@ if get_option('tests')
 
   tests = ['device-tree', 'display-panel', 'cutout', 'svg-path', 'timeout', 'utils']
   # These need data not available on the installed system:
-  not_installed = ['device-tree']
+  not_installed = ['test-device-tree']
 
   foreach test : tests
 
